### PR TITLE
Docs: New Version Updates

### PR DESCRIPTION
**Pull request by jfermi**
Change "Environment" to "Variable Set".
Update trace analyzer docs to include new config. Issue #2990 Update webui/transactions page to include new automate tab. Issue #2992

This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/docs/live-examples/opentelemetry-store/use-cases/user-purchasing-products.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ So in this case, we need to trigger four tests in sequence to achieve test the e
 
 ## Building a Transaction for This Scenario
 
-Using Tracetest, we can do that by [creating a test](../../../web-ui/creating-tests.md) for each step and later grouping these tests as [transactions](../../../web-ui/creating-transactions.md) that have an [environment](../../../concepts/environments.md)](../../../concepts/environments.md).
+Using Tracetest, we can do that by [creating a test](../../../web-ui/creating-tests.md) for each step and later grouping these tests as [transactions](../../../web-ui/creating-transactions.md) that have an [environment](../../../concepts/variable-sets.md)](../../../concepts/variable-sets.md).
```
Comment (kdhamric) [inline docs/docs/live-examples/opentelemetry-store/use-cases/user-purchasing-products.md:21]: You have the word 'environment' still here

File: docs/docs/live-examples/opentelemetry-store/use-cases/user-purchasing-products.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ So in this case, we need to trigger four tests in sequence to achieve test the e
...
  
 We can do that by creating the tests and transactions through the Web UI or using the CLI. In this example, we will use the CLI to create an environment and then create the transaction with all tests needed. The [assertions](../../../concepts/assertions.md) that we will check are the same for every single test.
```
Comment (kdhamric) [inline docs/docs/live-examples/opentelemetry-store/use-cases/user-purchasing-products.md:23]: The word environment is still here

File: docs/docs/web-ui/creating-environments.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 This page showcases how to create and edit environments in the Web UI.
 
 :::tip
-[To read more about environments check out environment concepts.](../concepts/environments.md)
+[To read more about environments check out environment concepts.](../concepts/variable-sets.md)
```
Comment (kdhamric) [inline docs/docs/web-ui/creating-environments.md:6]: environment still here

**Review by kdhamric — APPROVED** [review]
I noted a couple changes - should be good after those are done.

File: docs/docs/live-examples/opentelemetry-store/use-cases/user-purchasing-products.md (hunk 1 above)
Comment (jfermi) [reply docs/docs/live-examples/opentelemetry-store/use-cases/user-purchasing-products.md:21]: Thanks.

File: docs/docs/web-ui/creating-environments.md (hunk 3 above)
Comment (jfermi) [reply docs/docs/web-ui/creating-environments.md:6]: Thanks.

File: docs/docs/live-examples/opentelemetry-store/use-cases/user-purchasing-products.md (hunk 2 above)
Comment (jfermi) [inline docs/docs/live-examples/opentelemetry-store/use-cases/user-purchasing-products.md:23]:
```suggestion
We can do that by creating the tests and transactions through the Web UI or using the CLI. In this example, we will use the CLI to create a Variable Set and then create the transaction with all tests needed. The [assertions](../../../concepts/assertions.md) that we will check are the same for every single test.
```

File: docs/docs/live-examples/opentelemetry-store/use-cases/user-purchasing-products.md [hunk 4]
```diff
@@ -18,9 +18,9 @@ So in this case, we need to trigger four tests in sequence to achieve test the e
 
 ## Building a Transaction for This Scenario
 
-Using Tracetest, we can do that by [creating a test](../../../web-ui/creating-tests.md) for each step and later grouping these tests as [transactions](../../../web-ui/creating-transactions.md) that have an [environment](../../../concepts/environments.md)](../../../concepts/environments.md).
+Using Tracetest, we can do that by [creating a test](../../../web-ui/creating-tests.md) for each step and later grouping these tests as [transactions](../../../web-ui/creating-transactions.md) that have an [environment](../../../concepts/variable-sets.md)](../../../concepts/variable-sets.md).
```
Comment (jfermi) [inline docs/docs/live-examples/opentelemetry-store/use-cases/user-purchasing-products.md:21]:
```suggestion
Using Tracetest, we can do that by [creating a test](../../../web-ui/creating-tests.md) for each step and later grouping these tests as [transactions](../../../web-ui/creating-transactions.md) that have an [variable set](../../../concepts/variable-sets.md)](../../../concepts/variable-sets.md).
```

**Pull request merged**